airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -608,6 +610,51 @@ protected int getMaxRecordValueLimit() {
     return 1000000000;
   }
 
+  @Test
+  void testCustomDbtTransformations() throws Exception {
+    if (!implementsBasicNormalization()) {
+      return;
+    }
+
+    final JsonNode config = getConfigWithBasicNormalization();
+
+    final DbtTransformationRunner runner = new DbtTransformationRunner(pbf, NormalizationRunnerFactory.create(
+        getImageName(),
+        pbf,
+        config));
+    runner.start();
+    final Path transformationRoot = Files.createDirectories(jobRoot.resolve("transform"));
+    final OperatorDbt dbtConfig = new OperatorDbt()
+        .withGitRepoUrl("https://github.com/fishtown-analytics/jaffle_shop.git")
+        .withGitRepoBranch("main")
+        .withDockerImage("fishtownanalytics/dbt:0.19.1")
+        .withDbtArguments("debug");
+    if (!runner.run(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt debug Failed.");
+    }
+    dbtConfig.withDbtArguments("deps");
+    if (!runner.transform(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt deps Failed.");
+    }
+    dbtConfig.withDbtArguments("seed");
+    if (!runner.transform(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt seed Failed.");
+    }
+    dbtConfig.withDbtArguments("run");
+    if (!runner.transform(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt run Failed.");
+    }
+    dbtConfig.withDbtArguments("test");
+    if (!runner.transform(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt test Failed.");
+    }
+    dbtConfig.withDbtArguments("docs generate");
+    if (!runner.transform(JOB_ID, JOB_ATTEMPT, transformationRoot, config, dbtConfig)) {
+      throw new WorkerException("dbt docs generate Failed.");
```

Comment:
here, we only check this operations can be run - do you think it's valuable to check the contents? or would it be more cost than benefit?

Reply:
I think the `dbt test` command actually checks the content of the tables and would fail otherwise. You can read more here on the custom git repo:
https://github.com/fishtown-analytics/jaffle_shop

It might be worth to ensure in a test that failures are properly reported though... I will add that case

Reply:
that makes sense. can you drop a comment explaining that?